backend/remote-state/etcdv3/client.go

```diff
@@ -0,0 +1,195 @@
+package etcd
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	etcdv3 "github.com/coreos/etcd/clientv3"
+	etcdv3sync "github.com/coreos/etcd/clientv3/concurrency"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform/state"
+	"github.com/hashicorp/terraform/state/remote"
+)
+
+const (
+	lockAcquireTimeout = 2 * time.Second
+	lockInfoSuffix     = ".lockinfo"
+	lockSuffix         = ".lock"
```

Comment:
`lockSuffix` was strictly a consul implementation detail, so that the terraform lock works just like the consul cli locks. This may be different for etcd, or maybe it doesn't matter at all.

Reply:
Thanks 🙂 Removed in f60075f.